bigquery_etl/glam/generate.py

```diff
@@ -242,13 +250,15 @@ def main():
         table(
             "scalar_bucket_counts_v1",
             **models.scalar_bucket_counts(
-                source_table=f"glam_etl.{args.prefix}__clients_scalar_aggregates_v1"
+                source_table=f"glam_etl.{args.prefix}__clients_scalar_aggregates_v1",
+                **config[args.prefix],
```

Comment:
May I ask why this extra `**config[args.prefix]` was needed?

Reply:
The scalar_bucket_counts will need the minimum_client_count param from the generate.py